app.js

```diff
@@ -3,85 +3,107 @@ var correct = 0;
-} else {
-  alert('It\'s a yes or no question! Come on!');
+function question4() {
+  var answer4 = prompt('Is Logan\'s favorite food pizza?').toLowerCase();
+  if(answer4 === 'yes' || answer4 === 'y') {
+    alert('Nope! His favorite food is actually chinese food. And if you want to get specific, he loves General Tsao\'s Chicken!');
+  } else if(answer4 === 'no' || answer4 === 'n') {
+    alert('Pizza is really good, but you are right. It is not his favorite food, that would be chinese food, but it is a close second!');
+    correct += 1;
+  } else {
+    alert('It\'s a yes or no question! Come on!');
+  }
 }
 
-var logan = 0;
-while(logan != 24) {
-  var logan = parseInt(prompt('How old is Logan?'));
-  if(logan === 24) {
-    alert('Correct! The kids these days would probably call him old...');
+function question5() {
+  var answer5 = prompt('Is one of Logan\'s favorite hobbies watching anime?').toLowerCase();
+  if(answer5 === 'yes' || answer5 === 'y') {
+    alert('Indeed! If you are looking for suggestions, he recommends Sword Art Online or Clannad!');
     correct += 1;
-  } else if(logan > 24) {
-    alert('Yikes! Does he really look that old! Believe it or not, he is actually younger!');
-  } else if(logan < 24){
-    alert('That\'s generous of you to say, but no, he is older!');
+  } else if(answer5 === 'no' || answer5 === 'n') {
+    alert('Wrong! When he\'s not playing video games or doing his homework, he is watching a bunch of anime!');
   } else {
-    alert('Ah. We have a jokester here I see! Try a whole numerical number, that might work better!');
+    alert('It\'s a yes or no question! Come on!');
   }
 }
 
-var states = ['Arizona', 'California'];
-var tries = 0;
-while(tries < 6) {
-  var answer7 = prompt('Can you name a state Logan has lived in besides Washington?').toLowerCase();
-  for(var i = 0; i < states.length - 1; i++) {
-    if(answer7 === 'arizona' || answer7 === 'california') {
-      alert('You got it! Logan has lived in both Arizona and California in addition to Washington!');
+function question6() {
+  var logan = 0;
+  while(logan != 24) {
+    var logan = parseInt(prompt('How old is Logan?'));
+    if(logan === 24) {
+      alert('Correct! The kids these days would probably call him old...');
       correct += 1;
-      tries += 6;
+    } else if(logan > 24) {
+      alert('Yikes! Does he really look that old! Believe it or not, he is actually younger!');
+    } else if(logan < 24){
+      alert('That\'s generous of you to say, but no, he is older!');
     } else {
-      alert('Nope! I\'m sure that\'s a lovely state, but Logan has not lived there! Try again!');
-      tries += 1;
+      alert('Ah. We have a jokester here I see! Try a whole numerical number, that might work better!');
     }
   }
 }
 
+function question7() {
+  var states = ['Arizona', 'California'];
+  var tries = 0;
+  while(tries < 6) {
```

Comment:
@kaylynyuh add a new line here - just for cleanliness and delineation between your variable declarations and your while loop